src/GovLst.sol

```diff
@@ -370,11 +370,11 @@ abstract contract GovLst is IERC20, IERC20Metadata, IERC20Permit, Ownable, Multi
 
   /// @notice Returns the quantity of stake tokens that a given number of shares is valued at. In other words,
   /// ownership of a given number of shares translates to a claim on the quantity of stake tokens returned.
-  /// @param _amount The quantity of shares that will be converted to stake tokens.
```

Comment:
(Arbitrary comment)

It seems like a couple of the suggestions have to been implemented.

1. `_calcDepositId` gas optimization
2. Inconsistent nomenclature for UPDATE_DEPOSIT_TYPEHASH

Are we opting out of these changes?

Reply:
Good catches @alexkeating, I misunderstood the first one, thinking it was the same optimization in the override method I'd already implemented. The second I missed because I looked in the wrong file. Both are added now.

Reply:
Good catch, cleaned up